command/agent/fs_endpoint_test.go

```diff
@@ -106,6 +109,124 @@ func TestAllocDirFS_ReadAt_MissingParams(t *testing.T) {
 	})
 }
 
+func TestAllocDirFS_ACL(t *testing.T) {
+	t.Parallel()
+	assert := assert.New(t)
+
+	httpACLTest(t, nil, func(s *TestAgent) {
+		state := s.Agent.server.State()
+
+		req, err := http.NewRequest("GET", "/v1/client/fs/ls/", nil)
```

Comment:
If you can, could you just wrap this in a for loop that runs the same suite of tests against all endpoints:

```
for _, endpoint := range []string{"ls", "stat", ...} {
  go.Run(endpoint, func(t *testing.T) {
      req, err := http.NewRequest("GET", fmt.Sprintf("/v1/client/fs/%s/", endpoint), nil)
      ...
  })
}
```